x/auth/vesting/types/codec.go

```diff
@@ -18,6 +18,8 @@ func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&DelayedVestingAccount{}, "cosmos-sdk/DelayedVestingAccount", nil)
 	cdc.RegisterConcrete(&PeriodicVestingAccount{}, "cosmos-sdk/PeriodicVestingAccount", nil)
 	cdc.RegisterConcrete(&PermanentLockedAccount{}, "cosmos-sdk/PermanentLockedAccount", nil)
+	cdc.RegisterConcrete(&MsgCreateVestingAccount{}, "cosmos-sdk/MsgCreateVestingAccount", nil)
```

Comment:
Does this line mean there is no Amino JSON signing support for `MsgCreateVestingAccount` in 0.44 and 0.45?

Reply:
hmm, no, I think it would still work in 0.44 and 0.45, but be shown without the `type` and `value` fields, similarly to https://github.com/cosmos/cosmjs/issues/1026#issuecomment-1043165391